### DISCUSSION TITLE
How to Pictures / Albums Date and Time offset in bulk

### DISCUSSION THREAD

**Discussion by theleshek** [immich-app/immich#10235] [Q&A] [answered]
Hi there

It was time to move, I have Trunas Scale now and Immich on it. Following Guides ;)
I have imported now bunch of pictures on it already.

Now I have noticed that some pictures, especially those imported from cameras have incorrect dates, it usually seems to be an issue with dead battery and date in camera not reset after power on.

I know there is the way to set new date via Web UI, but it sets date, time and timezone same for all pictures.

**1. Is there a way to change that field on multiple assets  by offsetting existing value by set number of days and/or hours ?
2. Setting only common timezone for multiple pictures without changing datetime?**

For example (on picture) :
Date showend is 4 Jan 2006 08:02 TZ +01:00
Date to set:  4 Mar 2007 14:08 TZ +02:00

So actual offset for this **and following pictures** should be + 424 days, 6 hours and 6 minutes

Is it possible to do?
If yes, how (considering I'm new to all of it)?

![image](https://github.com/immich-app/immich/assets/55439893/fcf1e865-f639-458f-a327-dc1bedaeea56)

**Comment by flesser** [comment, marked as answer]
Currently its not possible yet (without a script or such using the API), but there's a feature request: https://github.com/immich-app/immich/discussions/9430
